keyboards/dz60/keymaps/billiams/README.md

```diff
@@ -0,0 +1,73 @@
+## Billiam's DZ60 layout
```

Comment:
Can you please rename this file so its name is fully lowercase?

e.g. `keyboards/dz60/keymaps/billiams/readme.md`

Reply:
done